cylc/flow/platforms.py

```diff
@@ -481,3 +481,9 @@ def get_random_platform_for_install_target(install_target):
     """Return a randomly selected platform (dict) for given install target."""
     platforms = get_all_platforms_for_install_target(install_target)
     return random.choice(platforms)
+
+
+def get_localhost_install_target():
+    """Returns the install target of localhost platform"""
+    localhost = get_platform('localhost')
```

Comment:
Is identical to:
```suggestion
    localhost = get_platform()
```

Reply:
I changed it, making the default of  `get_install_target_from_platform()` return localhost but then I changed it back as I think  `if platform['install target'] == get_localhost_install_target()` immediately makes more sense for future devs than `if platform['install target'] == get_install_target_from_platfrom() `, what do you think?

Reply:
It's fine. My suggestion only makes sense in context of platforms. Yours just makes sense.